tests/test_buscamin.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+import pytest
+
+from Subfunction_Module import buscamin
+
+
+def test_buscamin():
+    """
+    Test for the "buscamin" function;
+    Located in the Subfunction Module of py_pecg.
+    """
+
+    # Valid Cases
+    Test_Signal_1 = np.zeros(10, dtype=int)
+    assert buscamin(Test_Signal_1) == 0
+    Test_Signal_2 = np.linspace(1, 10, 10)
+    assert (
+        buscamin(Test_Signal_2) == 0
+    )  # No local minimum, only global minimum
```

Comment:
I think it's simpler as a one liner:
```suggestion
    assert buscamin(np.linspace(1, 10, 10)) == 0  # No local minimum, only global minimum
```